gslib/commands/acl.py

```diff
@@ -340,6 +346,50 @@ class AclCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    sub_command = self.args.pop(0)
+    if sub_command == 'get':
+      if StorageUrlFromString(self.args[0]).IsObject():
+        command_group = 'objects'
+      else:
+        command_group = 'buckets'
+      gcloud_storage_map = GcloudStorageMap(gcloud_command=[
+          'alpha', 'storage', command_group, 'describe', self.args[0], '--format=json[acl]'
+      ],
+                                            flag_map={})
+      self.args = self.args[:-1]
+    
+    elif sub_command == 'set':
+      if len(self.args[1:])>1:
+        url_list = self.args[1:]
+        object_or_bucket_url = [StorageUrlFromString(i) for i in url_list]
+      else:
+        url_list =[self.args[1]]
+        object_or_bucket_url = [StorageUrlFromString(self.args[1])]
+      if object_or_bucket_url[0].IsBucket():
+        command_group = 'buckets'
+      else:
+        command_group = 'objects'
+      if self.args[0] in CANNED_GCS_ACLS:
+        acl_flag= '--predefined-acl=' + self.args[0]
+      else:
+        acl_flag='--acl-file=' + self.args[0]
+      gcloud_storage_map = GcloudStorageMap(
+          gcloud_command=[
+              'alpha', 'storage', command_group, 'update'] + url_list +  [acl_flag],
+          flag_map={
+              '-a': GcloudStorageFlag('--all-versions'),
+              '-f': GcloudStorageFlag('--continue-on-error'),
+              '-R': GcloudStorageFlag('--recursive'),
+              '-r': GcloudStorageFlag('--recursive'),
+          })
+      
+      args_to_be_removed = len(url_list) + 1 
+      self.args = self.args[:-args_to_be_removed]
```

Comment:
Does this just leaves the flags? Might want to add a code comment, so people know what's left in `self.args` that the shim translates